specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7773,6 +7779,15 @@
       },
       "description": "Specifies the storage settings for the virtual machine disks."
     },
+    "SecurityProfile": {
+      "properties": {
+        "encryptionAtHost": {
+          "type": "boolean",
+          "description": "This property can be used by user in the request to enable or disable Host Encryption for the virtual machine or virtual machine scale set. This will enable the encryption for all the disks including Temp disk at host itself. <br><br> Default: The Encryption at host will be disabled unless this property is not set to true for the resource."
```

Comment:
Isn't the opposite true?   The Encryption at host will be disabled unless this property "is set to true" for the resource.